mitosheet/src/components/footer/SheetTab.tsx

```diff
@@ -58,49 +124,85 @@ export default function SheetTab(props: SheetTabProps): JSX.Element {
 
     return (
         <div 
-            className={classNames('tab', selectedClass)} 
+            className={classNames('tab', {'tab-graph': props.tabIDObj.tabType === 'graph'}, {'tab-selected': props.isSelectedTab})} 
             onClick={() => {
                 props.setUIState(prevUIState => {
-                    return {
-                        ...prevUIState,
-                        selectedSheetIndex: props.sheetIndex
+                    if (props.tabIDObj.tabType === 'data') {
```

Comment:
I think it's fine for now, but these if statements make me feel like there's an architectural change we could make that would make this a cleaner file; maybe the onClick or something is a prop? Idk. It's fine for now, just this code smells a bit.